lib/file-helper.unit.test.js

```diff
@@ -4,7 +4,7 @@ describe('getComponentData', () => {
   it('rejects if unable to load component data', async () => {
     await expect(fileHelper.getComponentData('not-a-real-component'))
       .rejects
-      .toThrow('Error: ENOENT: no such file or directory')
+      .toThrow(/ENOENT: no such file or directory/)
```

Comment:
We've removed a try/catch wrapping `new Error()` so the full error message is now coming through